File: level_1/약수의 개수와 덧셈.py
def solution(left, right):
    arr = [i for i in range(left, right+1)]
    count = 0
    for i in arr:
        imsy = []
        for j in range(1, i+1):
            if i % j == 0:
                imsy.append(j)
        if len(imsy) % 2 == 0:
            count += i
        else:
            count -= i

    return count

File: level_1/약수의 개수와 덧셈.java
class Solution {
    public int solution(int left, int right) {
        int answer = 0;
        for (int i = left; i <= right; i++) {
            int yaksu = 1;
            for (int j = 2; j <= i; j++) {
                if (i % j == 0)
                    yaksu++;
            }
            answer += (yaksu % 2 == 0) ? i : -i;
        }
        return answer;
    }
}